### ISSUE TITLE
6.26.14 tag fails to build

### ISSUE THREAD

**Issue by MrCarroll** [root-project/root#14138]
### Check duplicate issues.

- [X] Checked for duplicates

### Description

The 6.26.14 tag fails to build with the following errors emitted by GCC:
```
/root/parts/root/src/rootx/src/rootcoreteam.h:12:32: error: stray ‘\’ in program
   12 | const char *gROOTCoreTeam[] = {\n   "Rene Brun",
      |                                ^
/root/parts/root/src/rootx/src/rootcoreteam.h:31:29: error: stray ‘\’ in program
   31 |    "Enric Tejedor Saavedra",\n   0\n };
      |                             ^
/root/parts/root/src/rootx/src/rootcoreteam.h:31:35: error: stray ‘\’ in program
   31 |    "Enric Tejedor Saavedra",\n   0\n };
   ```
   ```
   /root/parts/root/src/rootx/src/rootcoreteam.h:12:33: error: ‘n’ was not declared in this scope
   12 | const char *gROOTCoreTeam[] = {\n   "Rene Brun",
      |                                 ^
/root/parts/root/src/rootx/src/rootcoreteam.h:12:37: error: expected ‘}’ before string constant
   12 | const char *gROOTCoreTeam[] = {\n   "Rene Brun",
      |                               ~     ^~~~~~~~~~~
/root/parts/root/src/rootx/src/rootcoreteam.h:34:1: error: expected declaration before ‘}’ token
   34 | } // namespace ROOT
      | ^
```

### Reproducer

Checkout 6-26-14 and attempt to build from source.

### ROOT version

6.26.14

### Installation method

Source

### Operating system

Ubuntu 20.04

### Additional context

Failing in Snap build environment, which could be related but the errors look like general syntax errors that would effect most build environments.

**Comment by guitargeek** [comment]
Hi! Is this not fixed by the following commit?
https://github.com/root-project/root/commit/41212a8ecd73920942a348b1167d7acf2d27f7a5

**Comment by MrCarroll** [comment]
Hi both,

Somewhat embarassingly, Danilo is correct. I've been occupied heavily with work lately and with this being my third build of ROOT today didn't notice the exact rookie mistake that's been presumed; I'd accidentally put into my build script the hash for 6.26.12 and not 6.26.14 which I intended (and more frustratingly, I'd then added a comment saying #6.26.14).

Please disregard this issue. It's correct in the context of 6.26.12 but then the problem was fixed already and 6.26.12 was only ever the latest release for about a day.

Thanks for your assistance both, and my apologies for the misfire!

**Comment by dpiparo** [comment]
@MrCarroll do not worry. These things happen, especially when one is under heavy load at work. Never hesitate to report issues as you encounter them, feedback is very important for us!
